cmd/minikube/cmd/service.go

```diff
@@ -77,11 +80,21 @@ var serviceCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
+		cfg, err := config.Load(profileName)
+		if err != nil {
+			exit.WithError("Error getting config", err)
+		}
+
 		urls, err := service.WaitForService(api, namespace, svc, serviceURLTemplate, serviceURLMode, https, wait, interval)
 		if err != nil {
 			exit.WithError("Error opening service", err)
 		}
 
+		if runtime.GOOS == "darwin" && cfg.Driver == oci.Docker {
+			out.FailureT("openning service in browser is not implemented yet for \"docker driver on mac\".\nPlease track the URL bellow to see updates for in progress work:\nhttps://github.com/kubernetes/minikube/issues/6778")
```

Comment:
"Opening service in browser is not implemented yet for docker driver on Mac.\nThe following issue is tracking the in progress work: